assets/stylesheets/test-details.scss

```diff
@@ -414,4 +414,14 @@ ul.modcategory {
     .fa-download:hover {
         opacity: inherit;
     }
+}
+
+.test-result-overview-link {
+    top: -0.1em;
+    position: relative;
+    left: 0.2em;
+    opacity: 0.8;
+    font-size:medium;
+    color: #446E9B;
+    text-decoration: underline dotted;
```

Comment:
This style has no effect but would look bad anyways so just remove the line.

Reply:
done